### PR TITLE
[AppService] az webapp list-runtimes: Fix the bug for list-runtimes

### LINKED ISSUE #12159: az webapp list-runtimes
## Describe the bug

**Command Name**
`az webapp list-runtimes`

**Errors:**
```
'bool' object has no attribute 'provider'
Traceback (most recent call last):
python3.6/site-packages/knack/cli.py, ln 206, in invoke
    cmd_result = self.invocation.execute(args)
cli/core/commands/__init__.py, ln 608, in execute
    raise ex
cli/core/commands/__init__.py, ln 666, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
cli/core/commands/__init__.py, ln 659, in _run_job
    six.reraise(*sys.exc_info())
...
cli/command_modules/appservice/custom.py, ln 704, in list_runtimes
    return [s['displayName'] for s in runtime_helper.stacks]
cli/command_modules/appservice/custom.py, ln 2159, in stacks
    self._load_stacks()
cli/command_modules/appservice/custom.py, ln 2180, in _load_stacks
    raw_stacks = self._client.provider.get_available_stacks(os_type_selected=os_type, raw=True)
AttributeError: 'bool' object has no attribute 'provider'
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az webapp list-runtimes`

## Environment Summary
```
Linux-4.15.0-1067-azure-x86_64-with-debian-stretch-sid
Python 3.6.5
Shell: bash

azure-cli 2.0.81

```
## Additional Context
I'm trying to run the example from https://docs.microsoft.com/en-us/azure/app-service/web-sites-php-configure

### PR DESCRIPTION
**Pull request by Juliehzl**
**History Notes:**  
(Fill in the following template if multiple notes are needed, otherwise PR title will be used for history note.)

[Component Name 1] (BREAKING CHANGE:) (az command:) make some customer-facing change.  
[Component Name 2] (BREAKING CHANGE:) (az command:) make some customer-facing change.

---
Fix #12159, #12160, #12164 

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py [hunk 1]
```diff
@@ -107,6 +107,9 @@ def test_webapp_e2e(self, resource_group):
             JMESPathCheck('httpsOnly', False),
         ])
 
+    def test_webapp_runtimes(self):
+        self.cmd('webapp list-runtimes')
```
Comment (panchagnula) [inline src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py]: Should we check that we get status 200 ?

Comment (qwordy) [reply src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py]: This test case is so simple. Maybe you can add some assertions.

Comment (Juliehzl) [reply src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py]: I just want to test the command will succeed. If the command fails the test will failed, too.

Comment (qwordy) [reply src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py]: For example, check the returned values, check their format, check not empty output

**Pull request merged**